### ISSUE TITLE
this.driver.sessionCapabilities() does not request remote inspector session

### ISSUE THREAD

**Issue by donmezburak** [appium/appium-desktop#1811]
## Appium or Appium Desktop?
- [x] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

Hello, I am Burak from robotic.mobi. We added a remote inspector feature. We have problem on line 214 Appium-method-handler.js:

`await this.driver.sessionCapabilities();`

This method does not call get session capabilities endpoint (GET /session/:session_id). Therefore, this returns null and remote inspector fails

<img width="1058" alt="Screen Shot 2021-05-22 at 12 42 38" src="https://user-images.githubusercontent.com/1921050/119222090-49a46400-bafb-11eb-8537-6aa09c6d27f8.png">

## Environment

- I am running Appium Desktop version 1.21 and on branch 1.21.
- I am on (pick one):
    Appium inspector on [x] Mac
    Appium backend server on [x] Linux

**Comment by jlipps** [comment]
Can you try the current version of the inspector, which you can get a beta release from at https://github.com/appium/appium-inspector/releases?

**Comment by donmezburak** [comment]
I downloaded dmg file and installed it. I am getting another error. Session is starting on our side however inspector shows error:

<img width="1392" alt="Screen Shot 2021-05-25 at 11 55 26" src="https://user-images.githubusercontent.com/1921050/119469637-3807ca00-bd50-11eb-9da1-420b73a325ed.png">


I tried to compile source code. Gui-common folder is empty. Same thing happens when pulling repo. `npm run build` gives this error:

![Screen Shot 2021-05-25 at 11 57 17](https://user-images.githubusercontent.com/1921050/119469789-5cfc3d00-bd50-11eb-8c86-10c5cef99a34.png)

Finally, I pulled the https://github.com/appium/appium-gui-libs repo also and build succeeded. However getting the same SSL error on image 1. By the way, I was getting the same SSL error on Appium-desktop repo on main branch. But, 1.21 branch was working and giving `await this.driver.sessionCapabilities();` null error.
